technical-reference/reference-technical-articles/technical-doc-token-converter.md

```diff
@@ -0,0 +1,226 @@
+    - [BTCBPrimeConverter](https://bscscan.com/address/0xE8CeAa79f082768f99266dFd208d665d2Dd18f53): accepting [BTCB](https://bscscan.com/address/0x7130d2A12B9BCbFAe4f2634d864A1Ee1Ce3Ead9c) and offering 0.5% of the interest reserves
+    - [ETHPrimeConverter](https://bscscan.com/address/0xca430B8A97Ea918fF634162acb0b731445B8195E): accepting [ETH](https://bscscan.com/address/0x2170Ed0880ac9A755fd29B2688956BD959F933F8) and offering 2.5% of the interest reserves
+- [VTreasury](https://bscscan.com/address/0xf322942f644a996a617bd29c16bd7d231d9f35e9)
+
+`XVSVaultConverter` and every Prime converter are instances of `SingleTokenConverter` contract.
+
+`RiskFundConverter` and `SingleTokenConverter` convert the incoming earnings into specific tokens, with `RiskFundConverter` converting them into RiskFund's convertible base asset and `SingleTokenConverters` converting them into their `baseAsset`. Each time a conversion gets complete, the resulting tokens are sent to their destination addresses, respectively.
+
+The following diagram illustrates the flow of funds between contracts and the architecture it has:
+
+<figure><img src="../../.gitbook/assets/token_converter_funds.svg" alt="Integration of Token converters in the Venus protocol"><figcaption>Integration of Token converters in the Venus protocol</figcaption></figure>
+
+- The transfer of funds to and from the `ProtocolShareReserve` contract are initiated by external agents (permissionless).
+- The transfers of funds from the Token converters to their destinations are performed on each conversion. The conversions are performed by external agents (permissionless).
+- The transfer of XVS from the `XVSVaultTreasury` to the `XVSVault` will be performed via VIP.
+
+`RiskFundConverter` and `SingleTokenConverters` possess the ability to "facilitate" token conversions for the income they receive, obtaining the desired tokens (base asset) by relying on oracle prices as reference points to accept conversions that external agents will carry out.
+
+Venus encourages these conversions by providing incentives that create arbitrage opportunities in the market, with the expectation that external agents will seize upon these opportunities.
+
+The diagram below illustrates the connection between income harvesting (the process by which Venus transfers income from its sources to the `ProtocolShareReserve`), distribution (application of the protocol's Tokenomics-defined rules), and conversion (acquiring the required base assets).
+
+<figure><img src="../../.gitbook/assets/token_converters_sequence.svg" alt="Diagram showing the relationship among the harvesting, distribution, and converting of income"><figcaption>Relationship among the harvesting, distribution, and converting of income</figcaption></figure>
+
+Regarding the destinations of the Token converters:
```

Comment:
```suggestion
Regarding the destinations of the Token Converters:
```

Reply:
I'm not sure about the exact level of heading but using headings instead of phrases with colons providers a better hierarchical organization

Reply:
Looks like there are no 2 level headings so could be 2